tests/overlay-local-store/remount.sh

```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/bin/sh
```
don't have `/usr/bin/env` in namespace. We can get bash by doing `remount.sh.in` (see other such files) but I don't think we need it for this script.